File: homeWork3/src/main/java/com/homework/homework3/MainActivity.java
package com.homework.homework3;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_photo_task5);
    }
}
//      Landscape and portrait:
//  activity_photo_constraint
//  activity_photo_linear
//  activity_poll_linear
//  activity_poll_table
//  activity_sign_in_linear
//  activity_sign_in_constraint
//  activity_android_linear

//      Only portrait:
//  activity_photo_relative
//  activity_sign_in_relative
//  activity_android_constraint
//  На задании 4,2 на activity_android_constraint до меня дошло, что
//  не один и тот же layout должен одонаково хорошо смотреться в гориз и верт располож, а
//  нужно делать отдельно layout-land со своей разметкой для горизонт расположения
//  о Боже, я три недели промучался, но я не сдавался =)

File: hw11/src/main/java/com/homework/hw11/MainActivity.kt
package com.homework.hw11

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.button.MaterialButton


class MainActivity : AppCompatActivity() {
    private lateinit var uploadContactsButton: MaterialButton

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        uploadContactsButton = findViewById(R.id.uploadContactsButton)

        uploadContactsButton.setOnClickListener {
            val intent = Intent(this, ContactListActivity::class.java)
            startActivity(intent)
        }
    }
}